### PR TITLE
Include the info on different environments

### PR DESCRIPTION
**Pull request by harshad16**
Include the info on different environments
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

## Related Issues and Dependencies

Related-to: https://github.com/thoth-station/thoth-application/issues/2477

### CODE REVIEW

File: docs/environments.md [hunk 1]
```diff
@@ -1,103 +1,57 @@
...
-- Deployment manifest can be found at: [Graph Manifest](../core/overlays/moc-prod/graph-prod)
-- we manage the storage via different project Thoth components which utlises the [thoth-storage](https://pypi.org/project/thoth-storages/) module. [Details](https://github.com/thoth-station/storages#thoth-storages)
-- Local setup and management information can be acquired at [thoth-storages](https://github.com/thoth-station/storages#thoth-storages)
 
-Extensions:
+## Semi-Prod Environments
 
-**amcheck**: we have setup amcheck extension for detecting database corruption errors.
-The [Issue](https://github.com/thoth-station/storages/issues/1586) based on which it was decided to use this extension.
+This environment is used as a temperory production, it is on the same standard as production in term of resources,
```
Comment (VannTen) [inline docs/environments.md:20]:
```suggestion
This environment is used as a temporary production, it is on the same standard as production in term of resources,
```

File: docs/environments.md [hunk 2]
```diff
@@ -1,103 +1,57 @@
 # Deployment Environments
 
-This section describes the different environments used to deploy all the Thoth Services to OpenShift Cluster.
+This section describes the different environments used for all the Thoth Services deployments on the OpenShift Cluster.
 
 *Note*: All environments can be deployed using ArgoCD or Kustomize.
 
+## Prod Environments
```
Comment (VannTen) [inline docs/environments.md:7]:
```suggestion
## Prod Environment
```
Might use the singular ? AFAICT it's one env right ?

File: docs/environments.md [hunk 3]
```diff
@@ -1,103 +1,57 @@
...
-Project Thoth uses Postgresql database for knowlegde graph storage.
+Server: [balrog-opf-aws](https://console-openshift-console.apps.balrog.aws.operate-first.cloud/)<br>
+Request access: Open PR to be included in [list of users](https://github.com/operate-first/apps/blob/master/cluster-scope/base/user.openshift.io/groups/thoth-devops/group.yaml)
 
-- Deployment manifest can be found at: [Graph Manifest](../core/overlays/moc-prod/graph-prod)
-- we manage the storage via different project Thoth components which utlises the [thoth-storage](https://pypi.org/project/thoth-storages/) module. [Details](https://github.com/thoth-station/storages#thoth-storages)
-- Local setup and management information can be acquired at [thoth-storages](https://github.com/thoth-station/storages#thoth-storages)
 
-Extensions:
+## Semi-Prod Environments
```
Comment (VannTen) [inline docs/environments.md:18]: Same as above

File: docs/environments.md [hunk 4]
```diff
@@ -1,103 +1,57 @@
...
-        Name        | Version |   Schema   |                        Description
---------------------+---------+------------+-----------------------------------------------------------
-amcheck            | 1.0     | public     | functions for verifying relation integrity
-pg_stat_statements | 1.6     | public     | track execution statistics of all SQL statements executed
-plpgsql            | 1.0     | pg_catalog | PL/pgSQL procedural language
-(3 rows)
-```
 
-**Note**: `amcheck functions may be used only by superusers.` we currectly have made our user the **superuser**, based on your setup either do the same or create different users, assign them priviledges, and set them up in pgbouncer.
+## Stage Environments
```
Comment (VannTen) [inline docs/environments.md:31]: Same as above

File: docs/environments.md [hunk 5]
```diff
@@ -1,103 +1,57 @@
...
-- [documentation/amcheck-next](https://access.crunchydata.com/documentation/amcheck-next/1.5/)
+Server: [ocp4-psi](https://ocp4.psi.redhat.com/)<br>
+Request access: Contact the Manager to be included in the [group](https://rover.redhat.com/groups/group/aicoe-thoth).
 
 ## Test Environments
 
-There might be multiple and personal test environment, but there is also one provided to the whole team to hack around. This test environment can be broken at any time, ArgoCD is not self-healing it or pruning unrelated resources. The test environment consists of one OpenShift project, uses an ephemeral PostgreSQL deployment inside of that project, and depends on an external Kafka Broker deployment.
-
+The Test environments are volatail and can be multiple at a give point of time, however devs can deploy there own if needed on the quicklab cluster (read here for more). The hosted test environment by team can be broken at any time, and ArgoCD is not set on self-healing or pruning of resources.<br> The test environment consists of one OpenShift project, uses an ephemeral PostgreSQL deployment inside of that project, and depends on an external Kafka Broker deployment.<br>
```
Comment (VannTen) [inline docs/environments.md:44]:
```suggestion
The Test environments are volatile and can be multiple at any given point in time, however devs can deploy their own if needed on the quicklab cluster (read here for more). The hosted test environment by team can be broken at any time, and ArgoCD is not set on self-healing or pruning of resources.<br>
The test environment consists of one OpenShift project, uses an ephemeral PostgreSQL deployment inside of that project, and depends on an external Kafka Broker deployment.<br>
```
spell.

Comment (VannTen) [reply docs/environments.md:44]: Also, what exactly does it mean that test environments can be multiple at any given in time ? Shouldn't we simply document what exists ?

File: docs/environments.md [hunk 6]
```diff
@@ -206,3 +182,55 @@ inspection-workflow | CPU limit | Mem limit
...
 inspection-build    | 1         | 1Gi
 inspection-run      | 2.5       | 2Gi
+
+
+## Database Setup
+
+Project Thoth uses Postgresql database for knowlegde graph storage.
+
+- Deployment manifest can be found at: [Graph Manifest](../core/overlays/moc-prod/graph-prod)
+- we manage the storage via different project Thoth components which utlises the [thoth-storage](https://pypi.org/project/thoth-storages/) module. [Details](https://github.com/thoth-station/storages#thoth-storages)
```
Comment (VannTen) [inline docs/environments.md:192]:
```suggestion
- we manage the storage via different project Thoth components which utlises the [thoth-storages](https://pypi.org/project/thoth-storages/) module. [Details](https://github.com/thoth-station/storages#thoth-storages)
```

**Review by VannTen — CHANGES_REQUESTED** [review]

**Comment by VannTen** [comment]
Something else I just thought about: it seems to me we often refer to environment by their "surname" (aka "balrog", "smaug" etc). Maybe we should include in the section titles those names ?
(Like
`## Prod Environment (aka Balrog)` etc

File: docs/environments.md [hunk 7]
```diff
@@ -1,103 +1,58 @@
...
-### MassOpenCloud
-
-On Mass Open Cloud Zero clusters
-
-Links:
-https://massopen.cloud/
-https://www.operate-first.cloud/
+Server: [ocp4-psi](https://ocp4.psi.redhat.com/)<br>
+Namespace: **thoth-test-core**<br>
+Request access: Contact the Manager to be included in the [group](https://rover.redhat.com/groups/group/aicoe-thoth).
```
[the remark this answers is not in the document]
Comment (VannTen) [reply docs/environments.md:56]: Not sure if several "Credentials" section would conflict though (on the anchor level I mean)

**Comment by codificat** [comment]
pre-commit failure (logging here in case logs disappear):

```
 trim trailing whitespace.................................................Failed
- hook id: trailing-whitespace
- exit code: 1
- files were modified by this hook
Fixing docs/environments.md
```

**Pull request closed without merging**